pavelib/assets.py

```diff
@@ -641,14 +641,28 @@ def restart_django_servers():
     ))
 
 
-def collect_assets(systems, settings):
+def collect_assets(systems, settings, debug=None, collectstatic_log=None):
     """
     Collect static assets, including Django pipeline processing.
     `systems` is a list of systems (e.g. 'lms' or 'studio' or both)
     `settings` is the Django settings module to use.
+    `debug` and `collectstatic_log` are used for determining where to
+       pipe collectstatic logging.
     """
+
+    # unless specified, collectstatic (which can be very verbose) pipes to /dev/null
+    collectstatic_stdout_str = "> /dev/null"
+    if debug:
+        # pipe to console
+        collectstatic_stdout_str = ""
+    if collectstatic_log:
+        # pipe to specified file
+        collectstatic_stdout_str = "> {output_file}".format(output_file=collectstatic_log)
```

Comment:
Yes, they do collide. My challenge here is accommodating 3 use cases:
1. When specifying debug (as a developer), pipe to console
2. When running bok-choy tests (as Jenkins, or as a developer), pipe to a default file location
3. When specifying a file location (as a developer), pipe to that location

When someone specifies *both* debug and a specific file location, I was figuring we use the file location, because I'm assuming only a developer would want both for specific reasons (i.e., let me see all the sass compilation in debug mode, but put all the collectstatic stuff somewhere else)

So, I am also trying to let the debug switch filter all the way into this method, but have bok-choy tests, which call the method as well, be able to specify something. Not sure if your suggestion will work for all those scenarios?